FirebaseCore/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # v6.6.0 -- M62
 - [changed] Reorganized directory structure.
+- [changed] [Firebase Installations](../FirebaseInstallations/CHANGELOG.md) throws an exception when there are missing required `FirebaseOptions` parameters (`APIKey`, `googleAppID`, and `projectID`). Please make sure your `GoogleServices-Info.plist` (or `FirebaseOptions` if you configure Firebase in code) are up to date. The file and settings can be downloaded from the [Firebase Console](https://console.firebase.google.com/). (#4683)
```

Comment:
I'm not sure this necessarily needs to be in the Core changelog, we can put it in the changelog online without having it mirrored here since it doesn't really belong in Core, we just want to make sure it's at the top level of the release notes online.

Reply:
SGTM if we can add it to devsite only. I'll remove it then